lib/tasks/import.rake

```diff
@@ -0,0 +1,20 @@
+require 'csv'
+
+namespace :import do
+  desc "Import data from CSV file"
+  task :members => :environment do 
+    
+
+    CSV.foreach("db/data/legislators-current.csv", headers: true) do |row|
+      member = Member.new
+      member.first_name = row['first_name']
+      member.last_name = row['last_name']
+      member.state = row['state']
+      member.district = row['district']
+      member.party = row['party']
+      member.address = row['address']
+      member.contact_form = row['contact_form']
+      member.save!
+    end
+  end
+end
```

Comment:
did you pull main into this branch? there should be more tasks for the categories and states csv files that were added. then you can add this info into that file and add :members to the "all" task array at the bottom!